scripts/check_lua_classes.py

```diff
@@ -22,7 +23,8 @@
           print("******* CHECK CLASS DECLARATIONS *******")
           problem_found = True
           print("Invalid/Improper Class Declarations Found:")
-        print("*" + print_root_regex.search(root).group(0) + "\\" + script + ":" + found_class)
+        path = print_root_regex.search(root).group(0)
+        print("*" + path + "\\" + script + ":" + found_class)
```

Comment:
Using str.format looks better:
``` python
print("*{}\\{}:{}".fprmat(path, script, found_class))
```

Reply:
Depending on what version of Python you want to target, there's f-string templates (3.6+):
```py
print(f"*{path}\\{script}:{found_class}")
```

Reply:
It would be a great addition, but 3.6 sounds a bit too new, my fedora which tends to deploy the newer versions doesn't even has 3.7 yet. There will be many users that use something older.

Reply:
The real condition here is that it has to run in travis_ci; which is currently running Ubuntu 14.04. I don't mind bringing in a back port of a newer python version; but if we don't have to it's easier.

Reply:
I believe that means 3.4.0 should be the target version.